sos/report/plugins/sunbeam.py

```diff
@@ -94,7 +94,8 @@ def setup(self):
 
             self.add_copy_spec([
                 f"{sb_snap_homedir}/*.log",
-                f"{sb_snap_homedir}/etc/*/*.log",
+                f"{sb_snap_homedir}/etc/*/*/*.log",
```

Comment:
We could use `**` here, e.g. `etc/**/*.log`, which matches any number of subdirs.